tools/packaging/kata-deploy/local-build/kata-deploy-binaries-in-docker.sh

```diff
@@ -37,7 +39,9 @@ if [ ! -d "$HOME/.docker" ]; then
 	remove_dot_docker_dir=true
 fi
 
-docker build -q -t build-kata-deploy \
+container_image="${CC_BUILDER_REGISTRY}:build-kata-deploy-$(get_last_modification ${script_dir}/dockerbuild/Dockerfile)"
```

Comment:
I didn't see the upcoming commits yet, so this comment might not apply: shouldn't `get_last_modification` return `${sha}-dirty` when the file is modified but not commited? Otherwise, if I want to test a change I did locally in the Dockerfile then I will need to necessarily make a commit.

Reply:
Hmm.  I didn't think about this case, as generally when I develop I do that committing things.

I can add a logic to have a `dirty`, no issues.